packages/nimble-components/src/rich-text/viewer/tests/rich-text-viewer.spec.ts

```diff
@@ -76,16 +76,16 @@ describe('RichTextViewer', () => {
         );
     });
 
-    it('set the markdown attribute and ensure the markdown property is not modified', () => {
+    it('set the markdown attribute and ensure the markdown property is modified', () => {
         element.setAttribute('markdown', '**markdown string**');
 
-        expect(element.markdown).toBe('');
+        expect(element.markdown).toBe('**markdown string**');
     });
 
-    it('set the markdown property and ensure there is no markdown attribute', () => {
+    it('set the markdown property and ensure there is markdown attribute', () => {
         element.markdown = '**markdown string**';
 
-        expect(element.hasAttribute('markdown')).toBeFalse();
+        expect(element.hasAttribute('markdown')).toBeTrue();
```

Comment:
With the above change I expect this should change to false as we will not serialize the property assignments back to the attribute,